Platform/ArrowlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -18,7 +18,7 @@
 #include <Library/GpioV2SiLib.h>
 #include <Library/PciePm.h>
 #include <Library/MtlSocInfoLib.h>
-#include <Library/P2SbLib.h>
+#include <Library/P2sbLib.h>
```

Comment:
Since you updated the header file in the #include statement, please rename the file Silicon/ArrowlakePkg/Include/Library/P2SbLib.h to P2sbLib.h